augur/housekeeper.py

```diff
@@ -336,3 +338,35 @@ def prep_jobs(self):
                 job['repos'] = rs
             # time.sleep(120)
 
+    def update_redirects(self, repo_group_id):
+        repos_urls = get_repos_urls(repo_group_id)
+        for url in repos_urls:
+            r = requests.get(url)
+            update_repo_url(url, r.url)
+
+    def get_value_from_fetch_data(data):
+        return data[0]
+
+    def get_repos_urls(self, repo_group_id):
+        repos_sql = s.sql.text("""
+                SELECT repo_git FROM repo
+                WHERE repo_group_id = '{}'
+            """.format(repo_group_id))
+
+        repos = self.db.execute(repos_sql).fetchall()
+
+        if len(repos) == 0:
+            logger.info("Did not find any repositories stored in augur_database for repo_group_id {}\n".format(repo_group_id))
+
+        repos_list = list(map(get_value_from_fetch_data, repos))
+
+        return repos_list
+
+    def update_repo_url(self, old_url, new_url):
+        update_sql = s.sql.text("""
+                UPDATE repo.repo_git SET VALUE = '{}
+                WHERE repo_git = '{}'
+            """.format(new_url, old_url))
+
+        repos = self.db.execute(update_sql)
```

Comment:
This looks like its going to take the revised URL reported by GitHub and make the change in the `repo` table when a repository is moved. Looks Quite Helpful and I'd like to test it out. @gabe-heim ? @ccarterlandis ?